packages/mco/components/drpolicy-list-page/drpolicy-list-page.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { pluralize } from '@odf/core/components/utils';
+import { getName } from '@odf/shared';
```

Comment:
```suggestion
import { getName } from '@odf/shared/selector';
```

Mostly this is the reason for your unit test failure

Reply:
yes, thank you